### ISSUE TITLE
Clarify what happened with the data on the official server

### ISSUE THREAD

**Issue by Sh4rK** [RobotGame/rgserver#48]
The readme currently only states that the server is no longer available. Can I get a clarification of what happened with all the data? Is it all deleted?

**Comment by WhiteHalmos** [comment]
That is correct, the database and all associated backups have been permanently deleted.

**Comment by WhiteHalmos** [comment]
Everything necessary for setting up a new server has been open sourced, so feel free to make your own. :)

**Comment by Sh4rK** [comment]
Thanks, I only asked to know whether my account still exists in some sense, or I can forget it (and delete it from my password manager), so that is answered. :)

You can close the issue if you want, or leave it open for others to see.

**Comment by WhiteHalmos** [comment]
Yep! That's precisely why I made sure to delete everything so users no longer have to worry about it. Will keep this one open, thanks for asking!
